shared/js/smart-screen/spatial_navigator.js

```diff
@@ -81,6 +117,14 @@
     _getRect: function snGetRect(elem) {
       var rect = null;
 
+      if (this.ignoreHiddenElement && elem instanceof HTMLElement) {
+        var computedStyle = window.getComputedStyle(elem);
+        if (computedStyle.getPropertyValue('visibility') == 'hidden' ||
+            computedStyle.getPropertyValue('display') == 'none') {
+          return null;
```

Comment:
Since `navigate` uses this function without any protect as well, we assume a user doesn't pass a hidden element inside collection as parameter of `navigate` or things go wrong. Maybe we can comment this case in `navigate`?